__tests__/index.test.js

```diff
@@ -194,3 +194,51 @@ describe('GET /api/articles',()=>{
 });
 
 
+describe('GET /apri/articles (topic query',()=>{
+    test('return a 200 code when a request is made to the api',()=>{
+        return request(app)
+        .get('/api/articles?topic=mitch')
+        .expect(200)
+    })
+    test('return a 200 code and allarticles filtered by query when a valid request is made to the api "mitch"',()=>{
```

Comment:
Remember that we aren't testing specific input/output scenarios we are doing 'integration testing' for our server and testing behaviours. We don't need to specify different queries, we can test one scenario and class this as a behaviour like: 'accepts a topic query'.